delete-user-data/extension.yaml

```diff
@@ -66,11 +66,12 @@ resources:
 params:
   - param: LOCATION
     type: select
-    label: Deployment location
+    label: Cloud Functions location
     description: >-
-      Where should the extension be deployed? You usually want a location close
-      to your database. For help selecting a location, refer to the [location
-      selection guide](https://firebase.google.com/docs/functions/locations).
+      Where do you want to deploy the functions created for this extension? 
+      You usually want a location close to your database. For help selecting a 
```

Comment:
close to your database or storage bucket, depending on what the user is using this extension for.

Reply:
done.